.github/workflows/linters.yml

```diff
@@ -0,0 +1,43 @@
+name: Linters check
+on: pull_request
+jobs:
+  linters_check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+            node-version: 12
+      - id: files
+        uses: jitterbit/get-changed-files@v1
+
+      - name: Install node modules and create a report folder
+        run: |
+          npm ci
+          for folders in `find -maxdepth 1 -name "cvat-*" -type d`; do
```

Comment:
I don't like `cvat-*` pattern. It isn't robust. Better to look for package.json file and get its directory.

Reply:
Redesigned the installation of node modules. It will be triggered if files with the required extension are found. To speed up the process.